src/renderer/containers/AddMetadataPage/index.tsx

```diff
@@ -72,66 +76,77 @@ export default function AddMetadataPage() {
     }, [dispatch, imagingSessions]);
 
     return (
-        <>
-            <div className={styles.contentContainer}>
-            {!isSelectedJobLoading && (
-            <>
-                {hasAttemptedSubmit && !appliedTemplate && (
-                <Alert
-                    className={styles.alert}
-                    message="Please select a template."
-                    type="error"
-                    showIcon={true}
-                    key="template-not-selected"
-                />
-                )}
-                <LabeledInput
-                className={styles.selector}
-                label={`Select Metadata ${SCHEMA_SYNONYM}`}
-                >
-                <TemplateSearch
-                    allowCreate={true}
-                    disabled={isTemplateLoading || isReadOnly}
-                    error={hasAttemptedSubmit && !appliedTemplate}
-                    value={appliedTemplate?.templateId}
-                    onSelect={(t) => dispatch(applyTemplate(t))}
-                />
-                </LabeledInput>
-            </>
-            )}
-            {isSelectedJobLoading ? (
-            <div className={styles.spinContainer}>
-                <div>Loading...</div>
-                <Spin />
+        <div>
+            <div>
+            {!selectedUploads.length && // If we're adding new files, not editing ones that have been uploaded.
+              <Button
+                className={styles.returnButton}
+                onClick={() => dispatch(selectPage(Page.UploadWithTemplate))}
+                type="ghost"
+              >
+                <span className={styles.returnButtonIcon}><ArrowLeftOutlined /></span> Add More Files to Template
+              </Button>
+            }
+            </div>
```

Comment:
Bit of a GitHub miss here - the above section is all that was added to this page. The rest was wrapped in another `<div>` and indented a bit.